ScheduleCreator/DBAdapter.java

```diff
@@ -16,21 +16,32 @@
  * application.
  *
  * @author Jamison Valentine, Ilyass Sfar, Nick Econopouly, Nathan Tolodzieki
- * 
- * Last Updated: 3/3/2020
+ *
+ * Last Updated: 3/6/2020
  */
 public class DBAdapter {
 
     protected static File selectedCourseFile = new File("src/ScheduleCreator/resources/raw/user_selected_courses.txt");
-
-    ;
+    //Regex expression to get day from the current format.
+    final static String getDay = "\\b([	](TR\\b|MW\\b|MWF\\b|M\\b|T\\b|W\\b|R\\b|F\\b|(TBA\\b)))\\b";
+    //Regex expression to get time from the current format.
+    final static String getTime = "(?<=([ ]\\b[A-Z]{3}\\b.\\b[0-9]{3}\\b.+ [0-9]{2}\\b)).+?(?=\\b((TR\\b|MW\\b|MWF\\b|M\\b|T\\b|W\\b|R\\b|F\\b|(	 	TBA\\b))))";
+    //Regex expression to get CRN from the current format.
+    final static String getCRN = "[0-9]{5}";
+    //Regex expression to get building from the current format.
+    final static String getBuilding = "(?<=\\b([	](TR\\b|MW\\b|MWF\\b|M\\b|T\\b|W\\b|R\\b|F\\b|(TBA\\b)))\\b).*(?=\\=)";
+    //Regex expression to get instructor from the current format.
+    final static String getInstructor = "(?<=\\=).*(?= )";
+    //Temporary holding place for course information while being parsed for requested information.
+    private static List<String> lines;
```

Comment:
Should this go in the function that uses it?

Reply:
Should it? I initially put it in the method that uses it, but I was trying to follow the style guide, I don't know.

Reply:
I feel like it should be a local variable, but it doesn't really matter.